### PR TITLE
Add clean task to gulp file

### PR DESCRIPTION
**Pull request by chuckh**
Add clean task to clean build directory, make part of default task.

### CODE REVIEW

**Comment by abdonrd** [comment]
Please, sort the `package.json` dependencies and LGTM! :)

File: gulpfile.js [hunk 1]
```diff
@@ -27,7 +28,12 @@ const fork = polymer.forkStream;
 let polymerJSON = require('./polymer.json');
 let project = new PolymerProject(polymerJSON);
 
-gulp.task('default', () => {
+// Clean build directory
+gulp.task('clean', () => {
+  return del(['build/**','!build']);
```
Comment (abdonrd) [inline gulpfile.js]: Why `'!build'`?

Comment (chuckh) [inline gulpfile.js]: Otherwise it will delete build folder also. See https://github.com/sindresorhus/del#beware

Comment (abdonrd) [inline gulpfile.js]: @chuckh But that's a problem?

Comment (chuckh) [inline gulpfile.js]: @abdonrd my initial testing not having it gave me some flaky results.  Did some more testing after all these changes and it seems to work fine without it. So I will remove it.

Comment (abdonrd) [inline gulpfile.js]: Nice! :)

**Comment by chuckh** [comment]
Closing in favor of PR #886.  Which includes this and updates for `polymer-build v0.3.0`.

**Pull request closed without merging**